### PR TITLE
Used new sheepit icons and done some background changes

### PR DESCRIPTION
**Pull request by eyeballcode**
Also fixed a bit of spelling stuff and times where a stream may not be closed.

The username and password fields in the settings activity also has a small space in between them

### CODE REVIEW

File: src/com/sheepit/client/os/Linux.java [hunk 1]
```diff
@@ -104,13 +105,15 @@ public int getMemory() {
...
 				}
 			}
-			scanner.close();
 		}
 		catch (java.lang.NoClassDefFoundError e) {
 			System.err.println("Machine::type error " + e + " mostly because Scanner class was introducted by Java 5 and you are running a lower version");
 		}
 		catch (Exception e) {
 			e.printStackTrace();
+		} finally {
```
Comment (laurent-clouet) [inline src/com/sheepit/client/os/Linux.java]: Please check the code formator provided.

**Comment by laurent-clouet** [comment]
This is a lot of stuffs going on here.
You did 3 commits but it should have been at least 5 to be more clear about what each change are for.

**Comment by laurent-clouet** [comment]
I do like the new title png but the icon is not visible enough.
That's why i didn't used the favicon of the website.
systray

instead maybe with black body it should be better 
sheepit-v5

**Comment by eyeballcode** [comment]
Maybe we should stick to the previous icon for the taskbar? My taskbar is dark so there's not really a way to have a win-win where everyone gets a nice icon

File: src/com/sheepit/client/os/Linux.java (hunk 1 above)
Comment (eyeballcode) [inline src/com/sheepit/client/os/Linux.java]: Ek eclipse still doesn't listen?

**Comment by laurent-clouet** [comment]
I didn't forget about your PR, i don't have much time these days.
I merged some typo, i'll do the rest soon.

**Comment by laurent-clouet** [comment]
I manually merged your grammar&typo on master (via c3e16af32f09ab8fd9dc67da20a51d71bef5707f ).

**Pull request closed without merging**